docs/logging-and-managing-experiment-results/updating-existing-experiment.rst

```diff
@@ -0,0 +1,139 @@
+.. _update-existing-experiment:
+
+Updating existing experiment
+============================
+[loom-placeholder]
+
+You can update experiments even after they finished running. This lets you add new data or visualizations to the previously closed experiment and makes multi-stage training convenient.
+
+.. _update-existing-experiment-basics:
+
+Why you may want to update an existing experiment?
+--------------------------------------------------
+Updating existing experiment can come in handy in several situations:
+
+* You want to add metrics or visualizations to the closed experiment.
+* You finished model training and closed the experiment earlier, but now you want to continue training from that moment. Actually, you can even make multiple iterations of the procedure: ``resume experiment -> log more data``. Have a look at the :ref:`simple example <update-existing-experiment-basics-simple-example>` below for details.
```

Comment:
This :ref: is to the very next section -> I'd drop it, feels a bit weird.

Reply:
ok